### DISCUSSION TITLE
NES BIOS file

### DISCUSSION THREAD

**Discussion by L00zr** [sanni/cartreader#879] [General Chat]
Hello, I am new to emulation and I recently got my first handheld (Powkiddy RGB30)(JELOS). I've been doing my best to follow along with retrogamecorps.com starter guide. My question is, if NES bios files are supposedly not necessary for emulation or were never a thing in the original consoles, then why are my first rom dumps of Super Mario Bros./Duck Hunt (USA) and Super Mario Bros. 2 (USA) loading EXTREMELY slow and weird? I used new MicroSD cards, plus I already tried formatting them then redumping roms anyway to try fixing the problem. I could maybe understand if only the Super Mario Bros./Duck Hunt did this since the pins were in real bad shape even after an aggressive cleaning with everything under the sun. But the motherboard and pins of my Super Mario Bros. 2 looked pristine, and even though I wiped them down with 99% isopropyl alcohol just in case, it also did not load correctly. What am I doing wrong? My other question is, if NES bios is supposedly not necessary or never existed originally, then why do i see a reference of "disksys.rom" in some places?

**Comment by Ancyker** [comment]
Did the CRCs match when dumping? If so, it's not an issue with the ROMs.

As for disksys.rom, it is for the Famicom Disk System:

![2023-05-01 23 11 11c](https://github.com/sanni/cartreader/assets/7105333/b6bddc19-edd6-4734-bfad-3db4f28f3792)

![2023-05-01 23 12 11c](https://github.com/sanni/cartreader/assets/7105333/8a6528e7-4307-4a85-aa04-b11b138c217a)

**Reply by L00zr** [reply]
What you wrote about the BIOS is a relief. As for the CRCs, I'm pretty sure they matched the references at https://nescartdb.com. Not sure what to do now. Last night someone on Reddit's r/RGB30 page told me that apparently JELOS has been acting up or whatever that means. They suggested I try switching to ARKOS in the mean time. Do you think that might fix my problem?

**Reply by Ancyker** [reply]
I'd say make sure it's up to date and if you are still having issues you'd probably have better luck asking in the JELOS Discord. I think most of the people in this community prefer original hardware or FPGA (Analogue Pocket, MiSTer, etc), so you might not get an answer. I do occasionally use emulation however I use a Steam Deck or a 3DS/PS Vita. I don't have much experience with the dedicated emulation devices.

**Reply by L00zr** [reply]
I see. Thank you so much!
